GOVERNANCE.md

```diff
@@ -1,131 +1,187 @@
+All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).
 
 ### Users
 Users are community members who have a need for the project. Through their usage, they give the project a purpose. Users are encouraged to participate in the project life by providing feedback on how their needs are satisfied.
 
 Users can help the project by:
 
 - Advertising and advocating for the project
-- Informing developers of strengths and weaknesses of the tool through the [user discussion list](https://forum.openrefine.org/) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)
+- Informing developers of the strengths and weaknesses of the tool through the [user discussion list](https://forum.openrefine.org/) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)
 - Providing moral support (a ‘thank you’ goes a long way)
-- Writing tutorials
+- Writing tutorials to contextualize OpenRefine for specific communities
 
 How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!
 
 ### Contributors
 Contributors are users getting involved in the project more closely. Contributions can take many forms:
 
 - Supporting new users via the [user discussion list](https://forum.openrefine.org/c/support/12)
-- Submitting patches to fix bugs or add features via pull requests
+- Submitting patches to fix bugs or add features via pull requests, comment on pull requests and help other contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8)
 - Maintaining and improving the [website](https://openrefine.org/)
 - Writing and maintaining the [documentation](https://github.com/OpenRefine/openrefine.org/tree/master/docs)
+- Helping [design interfaces](https://openrefine.org/docs/technical-reference/introduction) for OpenRefine
 
 How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)
 
 ### Committers
-If you make regular contribution to OpenRefine, you will most likely become a Committer.
+If you make regular contributions to OpenRefine, you will most likely become a Committer.
 
-Committers have earned enough trust from the community to review and merge pull requests.
+Committers have earned enough trust from the community to
 
-Therefore Committers:
-- Help contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8).
 - Triage issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
 - Have direct access to the code base to create new branches
-- Organize the wiki
-- Review and merge pull requests submitted by contributors
+- Organize the GitHub wiki
```

Comment:
This seems like it could be split up. Editing the wiki (which I thought had been retired) and triaging issues are less technically demanding tasks that don't need to be done by committers. Similarly, OpenRefine org membership seems like it could be decoupled from the type of contributor (although it may require adjustment to the privileges associated with it).

Reply:
The GitHub wiki still has several relevant documents (I did an audit when [preparing the contributor pathways](https://forum.openrefine.org/t/openrefine-2024-barcamp-improving-contributor-pathways/1630) and [listed them here](https://docs.google.com/document/d/1HIYA0fqOabjlxVPwdhqTpJD_0-TMs30Nk38pTGqJunE/edit?tab=t.0#heading=h.r3vv2g7bxk20)). I have been trying to migrate them to the documentation when possible. It is a lower priority and is not moving as fast as I wish. Due to the migration process, I think it is best to limit the wiki organization to committers (even if currently anyone with a GitHub account can edit). 

Currently, to add a label to issues, you need to be part of the GitHub organization (so be a committer).